### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeremyrader**
im leaving a comment on your pull request

### CODE REVIEW

**Comment by jeremyr8r** [comment]
I'm leaving another comment

**Comment by jeremyrader** [comment]
me too

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 Its a Monday...
 again
 Tomorrow is Tuesday
+And the day after is Wednesday
```
Comment (jeremyrader) [inline README.md]: That is correct...

File: README.md [hunk 2]
```diff
@@ -3,3 +3,4 @@
 Its a Monday...
 again
 Tomorrow is Tuesday
```
Comment (jeremyr8r) [inline README.md]: you are not wrong

**Pull request closed without merging**